packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3963,7 +3966,7 @@ export class ContainerRuntime
 			},
 			async (event) => {
 				this.verifyNotClosed();
-				const waitBlobsToAttach = props?.notifyImminentClosure;
+				this.imminentClosure = props?.notifyImminentClosure ?? this.imminentClosure;
```

Comment:
Currently this will never flip back to `false`, but might be worth adding some code to ensure that
```
if (!this.imminentClosure) ...
```